peerconnection.go

```diff
@@ -709,7 +709,7 @@ func (pc *PeerConnection) LocalDescription() *SessionDescription {
 	if localDescription := pc.PendingLocalDescription(); localDescription != nil {
```

Comment:
Above comment should be updated to match pion/webrtc's names.

Meaning of `pc.currentLocalDescription` and `pc.pendingLocalDescription` looks bit different from JavaScript one and is making the confusion.